src/sdk/namespace_fs.js

```diff
@@ -2282,11 +2284,19 @@ class NamespaceFS {
         return p.endsWith('/') ? p + config.NSFS_FOLDER_OBJECT_NAME : p;
     }
 
-    _get_file_md_path({ key }) {
+    async _get_file_md_path(fs_context, { key }) {
+        const p = this._get_file_path({ key });
+        // when the key refers to a directory (trailing /) with the disabled format (xattr are set on the dir)
```

Comment:
please move the comment into a jsdoc, it's hard to read the code when there are comments inside the function